src/CardArea/CardArea.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import CardArea from './CardArea';
+import { shallow } from 'enzyme';
+
+const userCardsData = [
+  { cardName: 'Blood Moon' }, 
+  { cardName: 'Breeding Pool' },
+  { cardName: 'Arid Mesa' }
+];
+const userDecks = [
+  { deckName: 'Ponza' },
+  { deckName: 'Burn' },
+  { deckName: 'Hollow One' }
+];
+const cardAreaView = 'myCardList';
+const setAsideViewMock = jest.fn(); 
+
+describe('CardArea', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = shallow(
+      <CardArea
+          userCardsData={userCardsData}
+          setAsideView={setAsideViewMock}
+          cardAreaView={cardAreaView}
+          userDecks={userDecks} />
+    );
+  });
+
+  it('should match the snapshot', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should sort card objects when sortCardsByName is called', () => {
+    const userCardsData = [
+      { cardName: 'Blood Moon' }, 
+      { cardName: 'Breeding Pool' },
+      { cardName: 'Arid Mesa' }
+    ];
```

Comment:
Hi Jeo! Were you not able to use the data on lines 5 - 8 for this test instead of having to redeclare it here?

Reply:
When I ran the test without it, the first expect was throwing an error because the array had already been sorted.